docs/modules/hdfs/examples/getting_started/hdfs.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: simple-hdfs
 spec:
   image:
-    productVersion: 3.4.0
+    productVersion: 3.3.6
```

Comment:
I think we want to keep getting started/demos as the new version

Reply:
Decided in Slack to move to 3.3.6